extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -23,6 +92,18 @@ export class AmpStoryShoppingAttachment extends AmpStoryPageAttachment {
     this.container_.textContent = this.myText_;
     this.element.appendChild(this.container_);
     applyFillContent(this.container_, /* replacedContent */ true);
```

Comment:
Please remove LN 91-94 and associated variables (this.container_, this.myText_).
They are not necessary.